web/src/containers/OrdersElement/index.tsx

```diff
@@ -143,33 +146,63 @@ class OrdersComponent extends React.PureComponent<Props, OrdersState>  {
+            <span style={{ color: setTradeColor(side).color }} key={id}>{orderSide}</span>,
+            <span key={id}>{orderType}</span>,
+            avg_price ? <Decimal key={id} fixed={currentMarket.price_precision} thousSep=",">{avg_price}</Decimal> : '-',
+            price ? <Decimal key={id} fixed={currentMarket.price_precision} thousSep=",">{price}</Decimal> : '-',
             <Decimal key={id} fixed={currentMarket.amount_precision} thousSep=",">{origin_volume}</Decimal>,
-            <Decimal key={id} fixed={currentMarket.amount_precision} thousSep=",">{executed_volume}</Decimal>,
-            <Decimal key={id} fixed={currentMarket.amount_precision} thousSep=",">{remaining_volume}</Decimal>,
-            <Decimal key={id} fixed={currentMarket.amount_precision} thousSep=",">{costRemaining.toString()}</Decimal>,
-            status,
-            state === 'wait' && <CloseIcon key={id} onClick={this.handleCancel(id)} />,
+            <Decimal key={id} fixed={currentMarket.amount_precision} thousSep=",">{total}</Decimal>,
+            <span key={id} className="split-lines f-small justify-content-end">
+                {trigger_price ? (
+                    <React.Fragment>
+                        <span>{this.props.intl.formatMessage({ id: 'page.body.trade.header.openOrders.lastPrice' })}</span>&nbsp;{getTriggerSign(ord_type, side)}&nbsp;&nbsp;
+                        <span style={{ color: setTradeColor(side).color }}>{Decimal.format(trigger_price, currentMarket.price_precision, ',')}</span>
+                    </React.Fragment>
+                ) : '-'}
+            </span>,
+            <span style={{ color: setTradeColor(side).color }} className="f-small" key={id}><Decimal fixed={2} thousSep=",">{+filled}</Decimal>%</span>,
+            <span key={id} className="f-small">{status}</span>,
+            (state === 'wait' || state === 'trigger_wait') && <CloseIcon key={id} onClick={this.handleCancel(id)} />,
         ];
     };
 
-    private getType = (side: string, orderType: string) => {
-        if (!side || !orderType) {
+    private getSide = (side: string) => {
+        if (!side) {
             return '';
         }
 
-        return this.props.intl.formatMessage({ id: `page.body.openOrders.header.orderType.${side}.${orderType}` });
+        return this.props.intl.formatMessage({ id: `page.body.openOrders.header.side.${side}` });
+    };
+
+    private getType = (orderType: string) => {
+        if (!orderType) {
+            return '';
+        }
+
+        return this.props.intl.formatMessage({ id: `page.body.trade.header.openOrders.content.type.${orderType}` });
+    };
+
+    private getPrice = (ord_type, status, avg_price, trigger_price, price) => {
+        if (ord_type === 'market' || status === 'done') {
+            return avg_price;
+        } else if (status === 'trigger_wait') {
+            return trigger_price;
+        } else {
+            return price;
+        }
     };
```

Comment:
I guess be better to create a helpers file for OrdersElement and move this function there